src/main/java/bot/Ui.java

```diff
@@ -81,8 +79,19 @@
             "Could not find local storage";
     public static final String THANKS_MESSAGE =
```

Comment:
Good naming conventions for final fields.

Reply:
Thank you for the review!